files/docker-override.conf

```diff
@@ -0,0 +1,4 @@
+[Service]
+EnvironmentFile=/run/docker-logging.env
+ExecStart=
+ExecStart=/usr/lib/coreos/dockerd daemon --host=fd:// $DOCKER_LOGGING_OPTS  $DOCKER_OPTS $DOCKER_CGROUPS $DOCKER_OPT_BIP $DOCKER_OPT_MTU $DOCKER_OPT_IPMAS
```

Comment:
I'd rather avoid overloading the ExecStart directive, as it might let us diverge from upstream without a clear way to come back from it.

Instead I think it'd be better to leverage the existing `Docker_OPTS` and to use Ansible's `assemble` module to create the corresponding `EnvironmentFile`, with the following structure:

```
DOCKER_OPTS=\
  --log-driver=json-file \

```

Note the use of backslashes to allow multiple actors to contribute to the content independently (the `vmware.flannel` role should be modified to do the same thing)